scalajvm/app/views/main.scala.html

```diff
@@ -1,4 +1,4 @@
-@(title: String, menu: Html = Html(""), scripts: Html = Html(""))(content: Html)
+@(title: String, menu: Html = Html(""), scripts: Html = Html(""), show_menus: Boolean = true)(content: Html)
```

Comment:
We use `camelCase` notation for parameters, please change `show_menus` to `showMenus`.